daemon/mgr/spec_mount.go

```diff
@@ -86,6 +91,10 @@ func setupMounts(ctx context.Context, c *Container, s *specs.Spec) error {
 
 		// TODO: support copy data.
 
+		if mp.Destination == "/dev/shm" && c.HostConfig.ShmSize != nil {
```

Comment:
Hi @Ace-Tang , I checked the swagger and found `If omitted, the system uses 64MB`. Is it correct for the API side? or it's just for the CLI?

Reply:
I think the swagger readme is copy from something where, and 64m is not implement in code

Reply:
OK. So could you mind to help us to update the description? 😄

Reply:
yeah....containerd sets it to 64M. 

https://github.com/containerd/containerd/blob/master/oci/spec_unix.go#L129-L133

So, we keep it on the swagger.